docs/assets/javascript/elements/uxdot-sidenav.ts

```diff
@@ -4,6 +4,8 @@ import { classMap } from 'lit/directives/class-map.js';
 import { customElement } from 'lit/decorators/custom-element.js';
 import { property } from 'lit/decorators/property.js';
 
+import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
```

Comment:
This change will be made against main https://github.com/RedHat-UX/red-hat-design-system/pull/1826

Reply:
#1826 is approved.